CRVResponse/test/calibration.fcl

```diff
@@ -1,103 +1,75 @@
 #
-# generate and simulate full CRV waveforms to find the pedestal
+# generate and simulate full CRV waveforms to reconstruct dark counts
 #
 #include "Offline/fcl/standardServices.fcl"
 #include "Offline/EventGenerator/fcl/prolog.fcl"
 #include "Production/JobConfig/common/prolog.fcl"
 #include "Production/JobConfig/primary/prolog.fcl"
 #include "Production/JobConfig/digitize/prolog.fcl"
 #include "Production/JobConfig/cosmic/prolog.fcl"
-#include "Production/JobConfig/reco/prolog.fcl"
 
-process_name : cosmicSimReco
+process_name : crvCalibSim
 
 source : {
   module_type :EmptyEvent
   firstRun: 1200
```

Comment:
That's the value that falls into the range for MDC2020.
Is that the intent?

Reply:
Yes, this calibration script is only used for testing purposes and won't be used in any production. I wanted to have a run number (I just changed it to 1210) close other run numbers for which we already have calibration constants in the database. The purpose was to test my GUI to show historic calibration constants and compare them with newly calculated calibration constants to automatically check, if these the calibration succeeded.